config/site/Rakefile

```
@@ -151,6 +151,30 @@ module ElasticGraph
           ::File.write(searchable_content_file, ::YAML.dump(searchable_content))
         end
 
+        desc "Extracts the site's documenation into one LLM-friendly file"
+        task extract_llm_txt_content: [:build_docs] do
+          require_relative "support/llm_content_extractor"
+
+          # Build Jekyll content so that we have parsable HTML to extract content from.
+          # (We couldn't figure out a way to extract the rendered content within Jekyll itself).
+          run_jekyll :build
+
+          output_file = SITE_SOURCE_DIR / "llms-full.txt"
+
+          extractor = ElasticGraph::LLMContentExtractor.new(
```

```suggestion
          extractor = LLMContentExtractor.new(
```

You're in the `ElasticGraph` module, so no need to namespace it.